docs/site/Authentication-overview.md

```diff
@@ -0,0 +1,68 @@
+---
+lang: en
+title: 'LoopBack 4 Authentication Overview'
+keywords: LoopBack 4.0, LoopBack 4, Authentication
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Authentication-overview.html
+---
+
+Security is of paramount importance when developing a web or mobile application
+and usually consists of two distinct pieces:
+
+- Authentication
+- Authorization
+
+Authentication is a process of verifying someone's identity before a protected
+resource is accessed.
+
+Authorization is a process of deciding if a user can perform an action on a
+protected resource.
+
+{% include note.html content=" For a description of an Authorization process, please see [Authorization](Loopback-component-authorization.md). " %}
+
+This document gives you an overview of the authentication system provided in
+LoopBack 4.
+
+Let's start with the following scenario: Suppose you want to secure endpoint
+`GET /todo` using
+[JWT(JSON web token)](https://github.com/auth0/node-jsonwebtoken#readme)
+strategy, the diagram below shows how such authentication process works with
+LoopBack's authentication mechanism.
+
+![authentication_overview_request_handle_flow](./imgs/authentication/authentication-overview.png)
+
+As you can see, a request with token in its header reaches the application. It's
+handled by the REST server's sequence, which invokes the authentication action
+to decode the user profile from token so that controller can inject the user
+into its functions. All you need to add is the code in programming colors:
```

Comment:
```suggestion
As illustrated above, a access token in Authorization header in making request to the route is 
handled by the REST server's sequence, which invokes the authentication action
to decode the user profile from token so that controller can inject the user to the system and perform the actions. For implementing this, All you need: is to add the code in programming colors:
```

Reply:
@madaky 👍 The controller doesn't execute action, I rephrased it to 

> As illustrated above, an access token in Authorization header in making request to the route is handled by the REST server's sequence, which invokes the authentication action to decode the user profile from token so that controllers can have the user injected.

> For implementing this, all you need to add is the code in programming colors: